ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource

```diff
@@ -10,9 +10,9 @@
 ${CODEFLARE-SDK_DIR}                     codeflare-sdk
 ${CODEFLARE-SDK_REPO_URL}                %{CODEFLARE-SDK_REPO_URL=https://github.com/project-codeflare/codeflare-sdk.git}
 ${DISTRIBUTED_WORKLOADS_RELEASE_ASSETS}  https://github.com/opendatahub-io/distributed-workloads/releases/latest/download
-${RAY_IMAGE}                             quay.io/modh/ray@sha256:0d715f92570a2997381b7cafc0e224cfa25323f18b9545acfd23bc2b71576d06
+${RAY_IMAGE}                             quay.io/modh/ray@sha256:db667df1bc437a7b0965e8031e905d3ab04b86390d764d120e05ea5a5c18d1b4
```

Comment:
@ChughShilpa As this image is related to 3.11 wondering this may fail with sdk e2e test as we are running sdk tests against python 3.9 version

Reply:
@jiripetrlik @sutaakar Thinking that we may need to run [sdk tests ](https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource#L54)against multiple python versions 3.9 and 3.11.. wdyt?

Reply:
Created jira https://issues.redhat.com/browse/RHOAIENG-15006 to work on this

Reply:
Yes right, updated the python version to 3.11 @Srihari1192

Reply:
Yes, it's good to run tests with both python versions. 
Since there is an ongoing discussion in [slack](https://redhat-internal.slack.com/archives/C055HEM0X7G/p1729688180583949?thread_ts=1729673578.283879&cid=C055HEM0X7G) for support of python 3.9 with codeflare-sdk 0.22, will do the change in another PR.
And if it is fine with you we can keep changes in this PR restricted to python 3.11 version @Srihari1192

Reply:
Apart from sdk tests, notebook image is used for distributed-workload tests also

Reply:
@ChristianZaccaria confirmed that the latest codeflare-sdk v0.22.0 will be in rhoai-2.15 in py3.11 images. Notebook images with py3.9 will use codeflare-sdk v0.21.1